Gems/EMotionFX/Code/Source/Integration/Editor/Components/EditorActorComponent.h

```diff
@@ -185,6 +187,13 @@ namespace EMotionFX
 
             ActorAsset::ActorInstancePtr        m_actorInstance;            ///< Live actor instance.
             AZStd::unique_ptr<RenderActorInstance> m_renderActorInstance;
+
+            AZ::Render::ModelReloadedEvent::Handler m_modelReloadedEventHandler{ [this](AZ::Data::Asset<AZ::RPI::ModelAsset> modelAsset)
```

Comment:
Personally I'd rather have this created at OnAssetUnloaded function when you needed. Keep the header clean.

Reply:
Sure I can move that.